asciidoc/courses/graph-data-science-fundamentals/modules/1-graph-algorithms/lessons/5-c-path-finding/questions/verify.adoc

```diff
@@ -14,7 +22,36 @@ input::answer[]
 [TIP,role=hint]
 .Hint
 ====
-You can use the same graph projection as the one created in the previous lesson. Make sure the relationships are projected with an `UNDIRECTED` orientation
+This is the Cypher you used in the previous lesson to find the shortest path between Kevin Bacon and Denzel Washington.
+
+[source, cypher, role=noplay]
+----
+// Create projection
+CALL gds.graph.project('proj',
+    ['Person','Movie'],
+    {
+        ACTED_IN:{orientation:'UNDIRECTED'},
+        DIRECTED:{orientation:'UNDIRECTED'}
+    }
+);
+
+// Find shortest path
+MATCH (kevin:Actor{name : 'Kevin Bacon'})
+MATCH (denzel:Actor{name : 'Denzel Washington'})
+
+CALL gds.shortestPath.dijkstra.stream(
+    'proj', 
+    {
+        sourceNode:kevin, 
+        TargetNode:denzel
+    }
+)
+
+YIELD sourceNode, targetNode, path
+RETURN sourceNode, targetNode, nodes(path) as path;
+----
+
+There are 4 relationship hops between Kevin Bacon and Denzel Washington.
```

Comment:
I don't think `hops` is helpful in this course at all

Reply:
I dont like "hops" either, its taken directly from the old course content, at the time I couldnt think of anything better.

 I think I will just change it to "relationships".